reporter/src/main/resources/templates/asciidoc/disclosure_document.ftl

```diff
@@ -31,7 +31,14 @@ Excluded projects and packages are ignored.
 :sectnums:
 :toc: preamble
 
-= Disclosure Document
+[#assign errorTitle = "DISCLAIMER! THERE ARE UNRESOLVED ISSUES IN THE ORT RESULT.
```

Comment:
This is certainly an obvious warning. Just wondering: would it make sense to include the issues themselves in the document, so the user can directly see what is still open? Maybe as a follow-up PR?

Reply:
Note that there easily might be hundreds of issues, as we're not limited to policy violations, so it might not feasible to list them unless you want to spend an appendix on it. Anyway, that's probably something we should consider for our internal template only.

Reply:
I'd drop " IN THE ORT RESULT" from the sentence.